validationframework-swing/src/main/java/com/google/code/validationframework/swing/decoration/IconComponentDecoration.java

```diff
@@ -218,6 +238,14 @@ public IconComponentDecoration(JComponent owner, AnchorLink anchorLinkWithOwner,
         decorationPainter.addMouseListener(toolTipVisibilityAdapter);
         decorationPainter.addMouseMotionListener(toolTipVisibilityAdapter);
         decorationPainter.addComponentListener(toolTipVisibilityAdapter);
+
+        scheduleTooltipUpdateTimer = new Timer(MINIMUM_TOOLTIP_VISIBLE_TIME_MS, new ActionListener() {
```

Comment:
Also, for more flexibility, it would be good to provide a constructor on which we can pass a delay (and then make `MINIMUM_TOOLTIP_VISIBLE_TIME_MS` public and rename it to `DEFAULT_MINIMUM_TOOLTIP_VISIBLE_TIME_MS` as done in several other classes in the framework).
And in case the developer passes 0 or a negative value, I would suggest not to instantiate any timer at all (and therefore have the original behavior).